bin/update

```diff
@@ -18,8 +18,9 @@ Dir.chdir APP_ROOT do
   puts "== Installing dependencies =="
 
   # Run bower in a thread and continue to do the non-js stuff
+  puts "Updating bower assets in parallel..."
```

Comment:
Uh... I just went with "assets" because we dump them into `vendor`/**`assets`**/`bower_components`, but "components" might be more correct.